Makefile

```diff
@@ -1,7 +1,29 @@
+scratch-gui-version = v3.2.37
+
 .PHONY: init
 init:
 	npm install --include=dev
 	brew install clang-format
+	make scratch-gui-install
+
+.PHONY: scratch-gui-install
+scratch-gui-install:
+	git clone --depth 1 --branch $(scratch-gui-version)  https://github.com/scratchfoundation/scratch-gui.git
+	cd scratch-gui && npm install
+	make scratch-gui-patch
```

Comment:
Maybe, it is better to leave the `patch` in a separate target so that the user is aware of the patch applied (and is it can also fail)